app/views/conversations/show.html.erb

```diff
@@ -138,18 +144,20 @@
                     keydown.enter->new-message#submitForm
                     keydown.shift+esc@document->new-message#focusKeydown
                     keydown.slash@document->new-message#focusKeydown
+                    keydown.esc->new-message#unfocusKeydown
```

Comment:
If you put this on your textarea instead of up here, do you still need to check the event.target and conditionally prevent default?

Reply:
Good call, I think you're right. I'll move it.